react/client/src/components/Textarea.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& .MuiTextField-root': {
+        margin: theme.spacing(1),
+      },
+      '& .MuiOutlinedInput-root': {
+        '& fieldset': {
+          borderRadius: '15px',
+          width: '20rem',
```

Comment:
Let's let this component be responsive and let the width resize based on its container. 

We may want to set a max-width in the future since I'll admit textareas look funky when larger.

Reply:
Great! I'll make that edit